app/api.js

```diff
@@ -0,0 +1,28 @@
+const wreck = require('@hapi/wreck')
+const config = require('./config')
+
+const get = async (url, token) => {
+  return wreck.get(`${config.statementsEndpoint}${url}`, getConfiguration(token))
+}
+
+const post = async (url, data, token) => {
+  const { payload } = await wreck.post(`${config.statementsEndpoint}${url}`, {
+    payload: data,
+    ...getConfiguration(token)
+  })
+  return payload
+}
+
+const getConfiguration = (token) => {
+  return {
+    headers: {
+      Authorization: token ?? ''
+    },
+    json: true
+  }
+}
+
+module.exports = {
+  get,
+  post
+}

```

Comment:
This file is only used in tests, so suggest moving it to the tests rather than the application code to avoid confusion.

I'm not sure the `statementsEndpoint` is necessary as a config value either because it's only used here and only effects tests so no value it being part of the Helm chart.